app/views/users/index.html.erb

```diff
@@ -1,7 +1,12 @@
-<h1>list of all users</h1>
-
 <ul>
   <% @users.each do |user| %>
-    <li><%= link_to user.name, user_path(user) %></li>
+    <li>
+      <a href="<%= user_posts_path(user) %>">
```

Comment:
- [ ] When a user is clicked on the users index page, the page should redirect to that user's show page, not their posts index page. Kindly fix this.